spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/JdbcAutoConfiguration.java

```diff
@@ -142,4 +146,33 @@ public JdbcSagaStore sagaStoreWithSchema(ConnectionProvider connectionProvider,
                             .serializer(serializer)
                             .build();
     }
+
+    @Bean
+    @ConditionalOnMissingBean
+    public DeadLetterSchema deadLetterSchema() {
+        return DeadLetterSchema.defaultSchema();
+    }
+
+    @Bean
+    @ConditionalOnMissingBean(name = "deadLetterQueueProviderConfigurerModule")
```

Comment:
Shouldn't this be conditional on missing bean of type?

Reply:
Ah, this is a left over from playing around...I'll adjust the annotations accordingly.